### PR TITLE
Update the real steps for AAD App Registration

### PR DESCRIPTION
**Pull request by ojrb**
This PR solved the Issue #3739, I add the real steps of the AAD App registration, based on the new options on the Azure portal. Please Review and Merge !

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: windows/security/threat-protection/windows-defender-atp/configure-mssp-support.md [hunk 1]
```diff
@@ -153,34 +153,28 @@ You'll need to create an application and grant it permissions to fetch alerts fr
...
-3. Click **New application registration**.
+3. Click **New registration**.
 
 4. Specify the following values:
 
     - Name: \<Tenant_name\> SIEM MSSP Connector (replace Tenant_name with the tenant display name)
-    - Application type: Web app / API
-    - Sign-on URL: `https://SiemMsspConnector`
+    - Supported account types: Account in this organizational directory only 
+    - Redirect URI: Select Web and type `https://<domain_name>/SiemMsspConnector`(replace <domain_name> with the tenant name)
```
Comment (nenonix) [inline windows/security/threat-protection/windows-defender-atp/configure-mssp-support.md]:
```suggestion
    - Redirect URL: Select Web and type `https://<domain_name>/SiemMsspConnector`(replace <domain_name> with the tenant name)
```

File: windows/security/threat-protection/windows-defender-atp/configure-mssp-support.md [hunk 2]
```diff
@@ -153,34 +153,28 @@ You'll need to create an application and grant it permissions to fetch alerts fr
...
 
-5. Click **Create**. The application is displayed in the list of applications you own.
+5. Click **Register**. The application is displayed in the list of applications you own.
 
-6. Select the application, then click **Settings** > **Properties**.
+6. Select the application, then click **Overview**.
 
-7. Copy the value from the **Application ID** field.
+7. Copy the value from the **Application (client) ID** field to a safe place, you will need this on the next step.
```
Comment (nenonix) [inline windows/security/threat-protection/windows-defender-atp/configure-mssp-support.md]:
```suggestion
7. Copy the value from the **Application (client) ID** field to a safe place, you will need this in the next step.
```

File: windows/security/threat-protection/windows-defender-atp/configure-mssp-support.md [hunk 3]
```diff
@@ -153,34 +153,28 @@ You'll need to create an application and grant it permissions to fetch alerts fr
...
-11. Click **Save**.
-
-12. Select **Keys** and specify the following values:
+9. Click **New client secret**.
 
     - Description: Enter a description for the key.
     - Expires: Select **In 1 year**
 
-13. Click **Save**. Save the value is a safe place, you'll need this 
+10. Click **Add**, copy the value of the client secret to a safe place, you will need this on the next step.
```
Comment (nenonix) [inline windows/security/threat-protection/windows-defender-atp/configure-mssp-support.md]:
```suggestion
10. Click **Add**, copy the value of the client secret to a safe place, you will need this in the next step.
```

**Comment by e0i** [comment]
@ojrb Please resolve copy editing suggestions. Thanks.

**Comment by ojrb** [comment]
Ready! accepted the copy-edit suggestions !

**Comment by e0i** [comment]
@mjcaparas The PR that addresses issue #3739 is available for review. Thanks.

**Comment by e0i** [comment]
@mjcaparas Following up for updates. Thanks.

**Comment by ojrb** [comment]
I correct the folder as @mjcaparas requested ! please review !

File: windows/security/threat-protection/microsoft-defender-atp/configure-mssp-support.md [hunk 4]
```diff
@@ -61,31 +69,36 @@ This action is taken by the MSSP. It allows MSSPs to fetch alerts using APIs.
...
 
 Granting access to guest user is done the same way as granting access to a user who is a member of your tenant. 
 
-If you're using basic permissions to access the portal, the guest user must be assigned a Security Administrator role in **your** tenant. For more information, see [Use basic permissions to access the portal](basic-permissions.md).
+If you're using basic permissions to access the portal, the guest user must be assigned a Security Administrator role in **your** tenant. For more information, see [Use basic permissions to access the portal](basic-permissions-windows-defender-advanced-threat-protection.md).
 
-If you're using role-based access control (RBAC), the guest user must be to added to the appropriate group or groups in **your** tenant. Fore more information on RBAC in Microsoft Defender ATP, see [Manage portal access using RBAC](rbac.md).
+If you're using role-based access control (RBAC), the guest user must be to added to the appropriate group or groups in **your** tenant. Fore more information on RBAC in Windows Defender ATP, see [Manage portal access using RBAC](rbac-windows-defender-advanced-threat-protection.md).
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/microsoft-defender-atp/configure-mssp-support.md]:
```suggestion
If you're using role-based access control (RBAC), the guest user must be added to the appropriate group or groups in **your** tenant. Fore more information on RBAC in Windows Defender ATP, see [Manage portal access using RBAC](rbac-windows-defender-advanced-threat-protection.md).
```

**Pull request merged**